Sources/PointFree/Auth.swift

```diff
@@ -62,14 +62,14 @@ extension URLRequest {
 
 public func readSessionCookieMiddleware<I, A>(
   _ conn: Conn<I, A>)
-  -> IO<Conn<I, Tuple2<A, Database.User?>>> {
+  -> IO<Conn<I, Tuple2<Database.User?, A>>> {
```

Comment:
The swap feels right but curious what drove it!

Reply:
nah nothing in particular I dont think